react_django/django-src/django_app/api.py

```diff
@@ -11,59 +11,65 @@ def str_to_bool(str):
 
 def create_user(request):
     if request.META['REQUEST_METHOD'] == 'POST':
+      try:
```

Comment:
Darryl, thanks for setting up error passing here. We're about to focus more on appropriate error-handling day-today.